utils/db.go

```diff
@@ -38,12 +40,43 @@ func openSQL(user, password, database string) (*sql.DB, error) {
 	}
 }
 
-// Convert is an internal function for Copy methods
-func Convert(redisType, sqluser, sqlpassword, sqldatabase, sqltable, redisaddr, redispass string, log bool) error {
-	db, err := openSQL(sqluser, sqlpassword, sqldatabase)
+// openPostgres opens a PostgreSQL connection with a desired user, password database name, host and port
+func openPostgres(user, password, database, host, port string) (*sql.DB, error) {
+	connectionString := fmt.Sprintf("postgres://%v:%v@%v:%v/%v?sslmode=disable", user, password, host, port, database)
```

Comment:
Change the Sprintf tags to strings (`%s`). I'll have more feedback tomorrow.

Reply:
interface tags (%v) have been changed to string tags (%s)